cli/js/error_stack.ts

```diff
@@ -1,7 +1,7 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 // Some of the code here is adapted directly from V8 and licensed under a BSD
 // style license available here: https://github.com/v8/v8/blob/24886f2d1c565287d33d71e4109a53bf0b54b75c/LICENSE.v8
-import * as colors from "./colors.ts";
+import * as colors from "../../std/fmt/colors.ts";
```

Comment:
undo please

Reply:
So I should instead implement colors as part of the cli crate?
If so, do I keep the `fmt/colors.ts` as a duplicate file or do I import the `cli/js/colors.ts` file to the `fmt`?

Reply:
Yes - there should be two separate implementations - as it is currently.